Userland/Libraries/LibWeb/Layout/BlockFormattingContext.cpp

```diff
@@ -154,13 +154,11 @@ void BlockFormattingContext::compute_width(Box const& box, AvailableSpace const&
         remaining_available_space.width = AvailableSize::make_definite(remaining_width);
     }
 
-    if (box_is_sized_as_replaced_element(box)) {
+    if (box_is_sized_as_replaced_element(box)) { //replace the below: we no longer need to cast, we can just call
```

Comment:
Comment obsolete

Reply:
Does this include "// FIXME: This should not be done by ReplacedBox"?

Reply:
Uhh, I'm not sure, I think no? Just the comment you added.